code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -304,7 +315,7 @@ namespace WijmoProvider.Feature {
                 );
                 this._triggerEventsFromColumn(
                     action.row,
-                    binding,
+                    action.col,
```

Comment:
We should be passing column uniqueId instead of column index.

Reply:
@JoaoFerreira-FrontEnd This is still missing, right?

Reply:
No, this is done